mabl snippets/elementRightClick.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Simulates right-clicking an element to display a context menu.
+ * Customize by adding a CSS selector to the element variable.
+ * To use an xPath, integrate the getElementByXpath snippet.
```

Comment:
To make it easier for the user to use this snippet, I think we should either provide a URL to the `getElementByXpath` snippet here in the doc, or include it as a function below. That way the user doesn't have to go looking for it (hopefully it will reduce the friction associated with JS snippets)

Reply:
Done